string node_name
int32 node_state
string extra_info